.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter"
+    },
+    "python.formatting.provider": "none"
+}
```

Comment:
Remove this, put this in gitignore.

This will be only for local.

then you can merge this.